### PR TITLE
rosetta code: factorial

### PR DESCRIPTION
**Pull request by renatamc**
Signed-off-by: Renata Carvalho <renatawm@gmail.com>

### CODE REVIEW

**Comment by privat** [comment]
+1

File: examples/rosettacode/factorial.nit [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# This program is public domain
+
+# Task: Factorial
+# SEE: <http://rosettacode.org/wiki/Factorial>
+
+module factorial
+
+redef class Int
+
+    fun fact: Int
```
Comment (Tagachi) [inline examples/rosettacode/factorial.nit]: Already exist. It's ```factorial``` in Int class.

Comment (privat) [inline examples/rosettacode/factorial.nit]: yes, but isn't the point of the task to show how to implement it? But I agree that something like the following should be added at the end of the file.

~~~nit
# By the way, `factorial` already exists in Int class
print n.factorial
~~~

File: examples/rosettacode/factorial.nit [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    do
+        if self < 1 then return 1
+        return self * (self-1).fact
+    end
+end
+var n = gets.to_i
+print n.fact
```
Comment (privat) [inline examples/rosettacode/factorial.nit]: I just see that there is a missing line-feed at the end of the file, git likes to complain about not having them. What text editor did you use already?

**Pull request merged**